packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp

```diff
@@ -142,7 +142,64 @@ void StepperIMEX_RK<Scalar>::setTableaus(std::string stepperType,
     this->setStepperType("IMEX RK 1st order");
     this->setOrder(1);
 
-  } else if (stepperType == "IMEX RK SSP2" || stepperType == "SSP2_222" ) {
+  } else if ( stepperType == "SSP1_111" ) 
+  {
+    {
+      // Explicit Tableau
+      typedef Teuchos::ScalarTraits<Scalar> ST;
+      const int NumStages = 1;
+      const int order = 1;
+      Teuchos::SerialDenseMatrix<int,Scalar> A(NumStages,NumStages);
+      Teuchos::SerialDenseVector<int,Scalar> b(NumStages);
+      Teuchos::SerialDenseVector<int,Scalar> c(NumStages);
+      const Scalar one = ST::one();
+      const Scalar zero = ST::zero();
+
+      // Fill A:
+      A(0,0) = zero; 
+
+      // Fill b:
+      b(0) = one; 
+
+      // Fill c:
+      c(0) = zero;
+
+
+      auto expTableau = Teuchos::rcp(new RKButcherTableau<Scalar>(
+        "Explicit Tableau - SSP1_111",
+        A,b,c,order,order,order));
+
+      this->setExplicitTableau(expTableau);
+    }
+    {
+      // Implicit Tableau
+      typedef Teuchos::ScalarTraits<Scalar> ST;
+      const int NumStages = 1;
+      const int order     = 1;
+      Teuchos::SerialDenseMatrix<int,Scalar> A(NumStages,NumStages);
+      Teuchos::SerialDenseVector<int,Scalar> b(NumStages);
+      Teuchos::SerialDenseVector<int,Scalar> c(NumStages);
+      const Scalar one = ST::one();
+
+      // Fill A:
+      A(0,0) =  one;
+
+      // Fill b:
+      b(0) = one;
+
+      // Fill c:
+      c(0) = one;
+
+      auto impTableau = Teuchos::rcp(new RKButcherTableau<Scalar>(
+        "Implicit Tableau - SSP1_111",
+        A,b,c,order,order,order));
+
+      this->setImplicitTableau(impTableau);
+    }
+    this->setStepperType("IMEX RK 1st order (SSP1_111)");
```

Comment:
@krcb This description shows that this scheme is a single stage method and not 2 stage

Reply:
Looks good